### DISCUSSION TITLE
Zepeto Script Package 1.4.2버전이 업데이트 되었습니다

### DISCUSSION THREAD

**Discussion by Zepeto-TechEvangelist** [naverz/zepeto-studio-kor#1081] [1. Announcements]
안녕하세요, https://github.com/naverz/zepeto-studio-kor/discussions/1035 이슈를 개선한 zepeto.script 패키지 1.4.2버전이 배포되어 안내 드립니다.

### [개선 사항]
* 스크립트 에러 검출 오류 수정

### [적용 방법]
Window → Package Manager 버튼을 눌러서 Package Manager를 오픈 → [ZEPETO.Script]를 누르고 오른쪽 하단의 Update 버튼을 누릅니다.
1.4.2 버전 업 후 컴파일러 재실행을 위해 유니티를 재시작 해 주세요.

<img width="1012" alt="스크린샷 2022-07-27 오후 2 11 38" src="https://user-images.githubusercontent.com/100118650/181166922-244173ed-031b-4ed2-9b98-297e5fc314fc.png">

감사합니다.

**Comment by koldshine** [comment]
안녕하세요. 1.4.2로 업데이트하고 나서 컴파일하면 unity에 아래와 같은 에러가 뜹니다
![111111111](https://user-images.githubusercontent.com/105689042/181400837-5e806204-1195-4ace-b44f-c10ac6be7a9d.png)

저 에러때문인지 1.4.2로 업데이트 이후 추가된 public 프로퍼티들이 inspector에 나타나지 않습니다.

**Reply by Zepeto-TechEvangelist** [reply]
안녕하세요, 월드 버전은 어떻게 되시나요?

**Reply by koldshine** [reply]
> 안녕하세요, 월드 버전은 어떻게 되시나요?

1.5.1 입니다

**Reply by Zepeto-TechEvangelist** [reply]
#1035 에서 작성해주신 예시 코드로는 컴파일 오류 재현이 되지 않고 있는데 
오류가 발생하는 스크립트와 인스펙터 스크린샷을 첨부해주시면 파악에 도움이 될 것 같습니다.

**Reply by koldshine** [reply]
> #1035 에서 작성해주신 예시 코드로는 컴파일 오류 재현이 되지 않고 있는데 오류가 발생하는 스크립트와 인스펙터 스크린샷을 첨부해주시면 파악에 도움이 될 것 같습니다.

Library 폴더를 삭제하고 프로젝트를 다시 여니 문제 없이 잘 됩니다.

**Reply by Zepeto-TechEvangelist** [reply]
넵 다행입니다~ 😄
